scripts/lib.sh

```diff
@@ -296,9 +295,23 @@ RUN pushd /root; \
 	make install > /dev/null 2>\&1; \
 	popd
 "
-	local musl_tar="musl-${MUSL_VERSION}.tar.gz"
-	local musl_dir="musl-${MUSL_VERSION}"
-	readonly install_musl="
+	# install musl for compiling rust-agent
+	install_musl=
+	if [ "${muslarch}" == "aarch64" ]; then
+		local musl_tar="${muslarch}-linux-musl-native.tgz"
+		local musl_dir="${muslarch}-linux-musl-native"
+		install_musl="
+RUN cd /tmp; \
+	curl -sLO https://musl.cc/${musl_tar}; tar -zxf ${musl_tar}; \
```

Comment:
This approach is consistent with the existing one, so no need to change it on this PR. However, I've raised https://github.com/kata-containers/osbuilder/issues/413 as we should be verifying the downloads too.

If you want to create a new commit to resolve that issue, great, but no obligation to on this PR! :smile:

Reply:
yep, I think the downloads support x86_64 too, but I haven't tried them out yet. 
So maybe when you tackle down the #413, you want to give it a try?? 😝